lib/chai/core/assertions.js

```diff
@@ -956,8 +950,25 @@ module.exports = function (chai, _) {
    *     expect('foo').to.have.length.within(2,4);
    *     expect([ 1, 2, 3 ]).to.have.length.within(2,4);
    *
+   * *Deprecatin notice:* Using `length` as an assertion will be deprecated
+   * in version 2.2.1 and removed in 2.3. Code using the old style of
```

Comment:
Next release will be `2.3.0`, and we'll have to remove it in `3.0.0` to keep semver. So could you please change this message to:

```js
   * *Deprecation notice:* Using `length` as an assertion will be deprecated
   * in version 2.3.0 and removed in 3.0.0. Code using the old style of
```

(Also you had a typo in `Depricatin`(sic))